sdk/core/azure-core/inc/http/http.hpp

```diff
@@ -50,8 +50,9 @@ class BodyBuffer
 
   uint8_t const* _bodyBuffer;
   uint64_t _bodyBufferSize;
+  uint64_t _currentPos;
   BodyBuffer(uint8_t const* bodyBuffer, uint64_t bodyBufferSize)
-      : _bodyBuffer(bodyBuffer), _bodyBufferSize(bodyBufferSize)
+      : _bodyBuffer(bodyBuffer), _bodyBufferSize(bodyBufferSize), _currentPos(0U)
```

Comment:
Don't need a U suffix for 0. Either it should be the right type for the variable you're initializing (in this case `UINT64_C(0)`), or omitted.